src/main/kotlin/com/avail/interpreter/primitive/general/P_PrintToErrorConsole.kt

```diff
@@ -52,8 +52,7 @@ import com.avail.io.TextOutputChannel
 
 /**
  * **Primitive:** Print the specified [string][StringDescriptor] to the
- * [current][Interpreter.fiber]'s [standard error channel][TextOutputChannel], [
- * ][ExecutionState.SUSPENDED] the current fiber until the string can be queued
+ * [current][Interpreter.fiber]'s [standard&#32;error&#32;channel][TextOutputChannel], [&#32;][ExecutionState.SUSPENDED] the current fiber until the string can be queued
```

Comment:
I just read your comment from an earlier file indicating you've reflowed these.  I'm going to continue scanning files within the earlier commit, since I'm at file 536/911.  I'll continue to note needed reflows, and if they're already resolved in your new commit, you can mark them as resolved.

Reply:
By the way, I found all occurrences of `* ]` and fixed those too. Reflowed as well.